README.md

```diff
@@ -27,7 +27,7 @@ Sample `.pre-commit-config.yaml`:
 
 ```yaml
 -   repo: https://github.com/shellcheck-py/shellcheck-py
-    rev: v0.8.0.4
+    rev: v0.9.0.1
```

Comment:
this will get updated when I release it -- this should be left alone so I can make a commit for the tag

Reply:
I'll just do this

Reply:
Removed in force-pushed 9f8327204d3b517630c9fce9694104d90392c1c3

Reply:
oh you got me, race condition :D